lib/Reader/readerir.cpp

```diff
@@ -2971,6 +2972,8 @@ FlowGraphNode *GenIR::fgSplitBlock(FlowGraphNode *Block, IRNode *Node) {
 }
 
 void GenIR::fgRemoveUnusedBlocks(FlowGraphNode *FgHead) {
+  bool IsOk = !verifyFunction(*Function, &dbgs());
+  assert(IsOk && "verification failed");
```

Comment:
Should any of this need to be under `#ifndef NDEBUG` / do you need a dummy use of IsOk to avoid release build warnings?